resourcehandler/k8sresources.go

```diff
@@ -68,6 +73,31 @@ func (k8sHandler *K8sResourceHandler) GetResources(frameworks []reporthandling.F
 	return k8sResourcesMap, allResources, nil
 }
 
+func getCloudProviderDescription(allResources map[string]workloadinterface.IMetadata, k8sResourcesMap *cautils.K8SResources) error {
+	if cloudsupport.IsRunningInCloudProvider() {
+		wl, err := cloudsupport.GetDescriptiveInfoFromCloudProvider()
+		if err != nil {
+			cluster := k8sinterface.GetCurrentContext().Cluster
+			provider := cloudsupport.GetCloudProvider(cluster)
+			// Return error with useful info on how to configure credentials for getting cloud provider info
+			switch provider {
```

Comment:
Why is the package not handling the error logic?

Reply:
It is, here we want to display custom error for developer and I thoght it's not supposed to be on the pkg. I can put it there tho